apps/webservice/src/app/api/v1/environments/route.ts

```diff
@@ -1,16 +1,20 @@
 import type { PermissionChecker } from "@ctrlplane/auth/utils";
 import type { User } from "@ctrlplane/db/schema";
-import type { z } from "zod";
 import { NextResponse } from "next/server";
+import { isPresent } from "ts-is-present";
+import { z } from "zod";
 
+import { takeFirst } from "@ctrlplane/db";
 import * as schema from "@ctrlplane/db/schema";
 import { Permission } from "@ctrlplane/validators/auth";
 
 import { authn, authz } from "../auth";
 import { parseBody } from "../body-parser";
 import { request } from "../middleware";
 
-const body = schema.createEnvironment;
+const body = schema.createEnvironment.extend({
+  expiresAt: z.coerce.date().optional(),
+});
```

Comment:
I like the suggestion for the min, but since you can have null epxiresAt which mens the environment lives forever, I don't think we really need a limit for the max